utils.js

```diff
@@ -214,17 +213,20 @@ module.exports = {
         })
     },
 
-    postMethodTo0box: function (url, data, clientId, public_key) {
+    postMethodTo0box: function (url, data, clientId, public_key,client_signature) {
+        if(client_signature){
+            headers_data['X-App-Client-Signature'] = client_signature
```

Comment:
I was looking over the 0box repo, and it seems like they use `X-App-Signature`, instead of the known standard.
I even opened [an issue](https://github.com/0chain/0box/issues/42) about it.

```suggestion
        if (client_signature) {
            headers_data['X-App-Signature'] = client_signature
```